src/config/schema.ts

```diff
@@ -112,8 +114,8 @@ export const configSchema = z
   .object({
     sponsorWalletMnemonic: z.string().refine((mnemonic) => ethers.utils.isValidMnemonic(mnemonic), 'Invalid mnemonic'),
     chains: chainsSchema,
-    deviationThresholdCoefficient: z.number().optional().default(1),
-    fetchInterval: z.number().positive(),
+    fetchInterval: z.number().positive(), // TODO: Rename to signedDataFetchInterval
+    deviationThresholdCoefficient: z.number(),
```

Comment:
hmm the previous field schema was requested [here](https://github.com/api3dao/airseeker-v2/pull/20#issuecomment-1750856113), perhaps we'd want to set them for the other fields too 🤔

Reply:
I see. I didn't know that. My view is that I prefer explicit over implicit for our configuration files (because every property matters). 

But because this was explicitly requested, I will revert this change.

Reply:
I think your point is valid, though, so perhaps they can be set to be explicit, but probably requires confirmation with the rest of the team 😄